src/v2/Components/FlashBanner/index.tsx

```diff
@@ -99,5 +100,7 @@ export const FlashBannerQueryRenderer: React.FC = () => {
         return <TrackedFlashBanner {...props} />
       }}
     />
+  ) : (
+    <TrackedFlashBanner />
```

Comment:
Does this component still need to render/be called (without props)?

I was thinking the first ( or one of the first lines in the `FlashBannerQueryRenderer` is `if (!user) return null` (and then everything else as-is).